pytential/symbolic/execution.py

```diff
@@ -83,7 +83,7 @@ def mesh_el_view(mesh, group_nr, global_array):
 
 
 class EvaluationMapperBase(PymbolicEvaluationMapper):
-    def __init__(self, bound_expr, actx: PyOpenCLArrayContext, context=None,
+    def __init__(self, bound_expr, actx: ArrayContext, context=None,
```

Comment:
That was intentional. It really only does support `PyOpenCLArrayContext` for now. Same below.

Hopefully a `sed -i s/actx: ArrayContext/actx: PyOpenCLArrayContext` can mostly take care of this.

Reply:
Yeah, got through about half of them and realized they're probably on purpose. Sorry for the noise!

Reverted in cdb0253.

Reply:
Np at all. Thanks for helping with this!